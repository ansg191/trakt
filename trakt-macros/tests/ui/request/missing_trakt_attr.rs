#[derive(Debug, Clone, trakt_macros::Request)]
pub struct Request {
    pub id: String,
}

fn main() {}
