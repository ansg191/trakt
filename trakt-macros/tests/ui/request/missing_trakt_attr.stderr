error: missing #[trakt(response = "...")] attribute
 --> tests/ui/request/missing_trakt_attr.rs:1:24
  |
1 | #[derive(Debug, Clone, trakt_macros::Request)]
  |                        ^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the derive macro `trakt_macros::Request` (in Nightly builds, run with -Z macro-backtrace for more info)
